modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -135,3 +135,39 @@ When you have your bootstrap image ready, you can deploy the image to the termin
 
 . Power on your POS terminals.
 . The branch server will bootstrap the terminals according to the data you have provided.
+
+
+
+== Re-Deploy Images to the Terminals
+
+You can instruct terminals to download and deploy images every time they are restarted.
```

Comment:
Replace `every time` by `when`.
This is nitpick, but just to be clear, below mentioned procedure are one time only. After terminal restart and redeployment/repartition the terminal is switched back to normal. Next restart is not forced to do anything.